doc/source/_toc.yml

```diff
@@ -40,6 +40,7 @@ parts:
         - file: ray-air/examples/upload_to_wandb
         - file: ray-air/examples/feast_example
         - file: ray-air/examples/automl_with_ray_air
+        - file: ray-air/examples/batch_tuning
```

Comment:
Please set SMOKE_TEST = True and then put the ipynb on the BUILD file so that we run it in CI.